### DISCUSSION TITLE
Validatation of slug values

### DISCUSSION THREAD

**Discussion by nschonni** [mdn/mdn-community#180] [Platform] [answered]
Currently there was one case of invalid values crashing the parser as discussed in https://github.com/mdn/content/pull/329#issuecomment-747699297

My 2 cents is that is should probably be an allow-list following the URL RFC like https://stackoverflow.com/a/1547940/455535 rather than a deny-list of characters.

There is probably a separate discussion needed for valid file paths so that the repo can be checked out on both unix and Windows filesystems

**Comment by Ryuno-Ki** [comment]
Sounds like a good idea.
I'd vote to ban at the very least Unicode confusables.

**Comment by bsmth** [comment, marked as answer]
I think most of this has been resolved now, there's another discussion about this here: https://github.com/orgs/mdn/discussions/293

Thank you :)
